test/suites/para/test_tanssi_containers.ts

```diff
@@ -341,35 +344,88 @@ describeSuite({
 
         it({
             id: "T16",
-            title: "Count number of tanssi collators before, during, and after 2002 chain",
-            timeout: 150000,
+            title: "Count number of tanssi collators before and during 2002 chain",
             test: async function () {
                 // This test depends on T12 and T15 to set blockNumber2002Start and blockNumber2002End
-                // TODO: don't hardcode the period here
-                const sessionPeriod = 10;
                 // The block range must start and end on session boundaries
                 expect(blockNumber2002Start % sessionPeriod).to.be.equal(0);
                 expect(blockNumber2002End % sessionPeriod).to.be.equal(0);
                 expect(sessionPeriod < blockNumber2002Start).to.be.true;
                 expect(blockNumber2002Start < blockNumber2002End).to.be.true;
-                // Start from block 5 because block 0 has no author
-                const blockNumber = sessionPeriod;
-                // Before 2002 registration: 4 authors
-                // TODO: this passes if only 2 authors are creating blocks, think a way to test that case
-                await countUniqueBlockAuthors(paraApi, blockNumber, blockNumber2002Start - 1, 4);
-
-                expect(sessionPeriod * 5 < blockNumber2002End, "2002 should have deregistered after first rotation");
-                expect(sessionPeriod * 10 > blockNumber2002End, "2002 should have deregistered before second rotation");
-
-                // While 2002 is live: 2 authors (the other 2 went to container chain 2002)
-                // We take from the first block that rotates, otherwise rotation kicks in
-                await countUniqueBlockAuthors(paraApi, sessionPeriod * 10, blockNumber2002End - 1, 2);
+                const fullRotationBlock = 50;
```

Comment:
I think we can do this in a more parametric manner. Meaning that we can always do `blockNumber2002Start%blocksPerSesssion == 0`

Reply:
What do you mean?

Reply:
here you are comparing your tests cases against a hardcoded block. But maybe it would be good to compare it against the next full rotation block (regadless if its 50 or 100)

Reply:
it's not very important as 99.5% of the times this block willl be 50, but if it's easy to change it maybe it's worth doing

Reply:
I'm a bit scared but will see if it can be done

Reply:
no worries, thanks for trying!